index.js

```diff
@@ -11,7 +11,9 @@ var candidateSelectors = [
 ];
 var candidateSelector = candidateSelectors.join(',');
 
-var matches = Element.prototype.matches || Element.prototype.msMatchesSelector || Element.prototype.webkitMatchesSelector;
+var matches = typeof Element === 'undefined'
+  ? function () {}
+  : Element.prototype.matches || Element.prototype.msMatchesSelector || Element.prototype.webkitMatchesSelector;
```

Comment:
Do you think it's preferable to do this or to much `matches` within the scope of the `tabbable` function and add a `if (typeof window === 'undefined') return` at the top of that function's scope?

Reply:
I think that would be perfectly fine too. No preference here. 🙂